pyproject.toml

```diff
@@ -17,6 +17,7 @@ pydantic = ">=2.0.0,<3.0.0"
 pydantic-settings = ">=2.0.0,<3.0.0"
 requests = "^2.32.0"
 PyJWT = "^2.8.0"
+bcrypt = "<4.0.0"
```

Comment:
Can you elaborate on this one? (it's already in the passlib extra) did you encounter a bug with the lockfile?